manifests/service.pp

```diff
@@ -1,3 +1,5 @@
+# @api private 
+# This class handles the ntp service. Avoid modifying private classes.
 #
```

Comment:
Absurdly small nitpick, I think we should remove this blank comment line so our formatting is consistent across all the classes